src/screens/discover/components/DiscoverHome.js

```diff
@@ -18,11 +19,13 @@ import {
   backupsCard,
   cryptoAndWalletsCard,
 } from '@/components/cards/utils/constants';
+import { OpRewardsCard } from '@/components/cards/OpRewardsCard';
 
 export default function DiscoverHome() {
   const { network } = useAccountSettings();
   const accountAsset = useAccountAsset(ETH_ADDRESS);
   const profilesEnabled = useExperimentalFlag(PROFILES);
+  const optimismRewardsEnabled = useExperimentalFlag(OP_REWARDS);
```

Comment:
thinking out loud but we should remote config this

Reply:
Yeah we probably should, it will be easy to replace though and it will be easier to develop this way